src/CwpSearchIndex.php

```diff
@@ -27,6 +27,11 @@ abstract class CwpSearchIndex extends SolrIndex
         '_text',
         '_spellcheckText',
     ];
+    
+    private static $casting = [
+        'getFieldDefinitions' => 'HTMLText',
+        'FieldDefinitions' => 'HTMLText'
```

Comment:
Do we need both of these? I would've expected `$index->getFieldDefinitions()` to use `FieldDefinitions` for casting, I suppose